### PR TITLE
Minor updates to plot.ly/api

### PR DESCRIPTION
**Pull request by jdamiba**
closes https://github.com/plotly/streambed/issues/11343

changed API Libraries to Open Source Graphing Libraries throughout. 

added a box for dash and for REST API. 

Removed API Protocol and Excel menus. 

Changed Plotly Workshop to DASH DAQ. 

changed footers to match plot.ly : closes https://github.com/plotly/documentation/issues/1055

### CODE REVIEW

**Comment by jdamiba** [comment]
Outstanding question: which images should I use for the REST API box?

**Comment by jackparmer** [comment]
This one looks pretty good in terms of matching the others and being clean:

https://cdn-images-1.medium.com/max/1200/1*uHzooF1EtgcKn9_XiSST4w.png

**Comment by Kully** [comment]
Going to give this a check now

**Comment by Kully** [comment]
The REST icon is too small. Can you make it bigger so that it matches the size of the other icons around it?

screen shot 2018-08-29 at 12 28 50 pm

Also replace the dash icon you are using with https://github.com/plotly/design-assets/blob/master/logos/dash/images/dash-logo-avatar.png. It is the standard avatar for dash and consistent with the icon used in the page it links to.

**Comment by Kully** [comment]
Not blocking 🚫 , but I think putting `Python`, `R` and `Others` sections on one line would look better in my opinion. All that extra white space on the Others section just looks weird to me.

screen shot 2018-08-29 at 12 51 16 pm

File: _posts/python/3d/2017-02-24-python-3d-index.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
 		<div class="--title">
 			<div class="--category-img"><img src="https://plot.ly/gh-pages/documentation/static/images/python-small.png" alt=""></div>
 			<div class="--body">
-				<h1>Plotly Python Library 3D Charts</h1>
+				<h1>Plotly Python Open Source Library 3D Charts</h1>
```
Comment (Kully) [inline _posts/python/3d/2017-02-24-python-3d-index.html]: should this not be `Open Source Graphing Library` to be consistent?

File: _posts/python/basic/2017-02-24-python-basic-index.html [hunk 2]
```diff
@@ -19,7 +19,7 @@
 		<div class="--title">
 			<div class="--category-img"><img src="https://plot.ly/gh-pages/documentation/static/images/python-small.png" alt=""></div>
 			<div class="--body">
-				<h1>Plotly Python Library Basic Charts</h1>
+				<h1>Plotly Python Open Source Library Basic Charts</h1>
```
Comment (Kully) [inline _posts/python/basic/2017-02-24-python-basic-index.html]: same here as ^

File: _posts/python/fundamentals/2017-02-24-python-fundamentals-index.html [hunk 3]
```diff
@@ -19,7 +19,7 @@
 		<div class="--title">
 			<div class="--category-img"><img src="https://plot.ly/gh-pages/documentation/static/images/python-small.png" alt=""></div>
 			<div class="--body">
-				<h1>Plotly Python Library Fundamentals</h1>
+				<h1>Plotly Python Open Source Library Fundamentals</h1>
```
Comment (Kully) [inline _posts/python/fundamentals/2017-02-24-python-fundamentals-index.html]: same here: ...Graphing Library

File: _posts/python/fundamentals/config/2015-06-30-config_opts.html [hunk 4]
```diff
@@ -31,7 +31,7 @@ <h4 id="New-to-Plotly?">New to Plotly?<a class="anchor-link" href="#New-to-Plotl
 </div>
 <div class="inner_cell">
 <div class="text_cell_render border-box-sizing rendered_html">
-<h4 id="Online-Configuration-Options">Online Configuration Options<a class="anchor-link" href="#Online-Configuration-Options">&#194;&#182;</a></h4><p>Config options set via our API libraries are overridden on graphs hosted on plot.ly (i.e. when working online).
+<h4 id="Online-Configuration-Options">Online Configuration Options<a class="anchor-link" href="#Online-Configuration-Options">&#194;&#182;</a></h4><p>Config options set via our Open Source Graphing libraries are overridden on graphs hosted on plot.ly (i.e. when working online).
```
Comment (Kully) [inline _posts/python/fundamentals/config/2015-06-30-config_opts.html]: capitalize `libraries` -> `Libraries` for consistency

File: _posts/python/scientific/2017-02-24-python-scientific-index.html [hunk 5]
```diff
@@ -19,7 +19,7 @@
 		<div class="--title">
 			<div class="--category-img"><img src="https://plot.ly/gh-pages/documentation/static/images/python-small.png" alt=""></div>
 			<div class="--body">
-				<h1>Plotly Python Library Scientific Charts</h1>
+				<h1>Plotly Python Open Source Library Scientific Charts</h1>
```
Comment (Kully) [inline _posts/python/scientific/2017-02-24-python-scientific-index.html]: graphing library

File: all_static/images/Plotly-Feed.html [hunk 6]
```diff
@@ -12381,7 +12381,7 @@ <h4 class="delta heading text--left">The Plotly community</h4>
                 </div><!--
                 --><div class="grid__item one-half">
                     <h4 class="delta heading text--left">Programmer? We've got you covered.</h4>
-                    See our <a href="https://plot.ly/api/" target="_self" class="link link--impt epsilon">API libraries</a><br>
+                    See our <a href="https://plot.ly/api/" target="_self" class="link link--impt epsilon">Open Source Graphing libraries</a><br>
```
Comment (Kully) [inline all_static/images/Plotly-Feed.html]: capital L on libraries

File: _posts/python/statistical/2017-02-24-python-statistic-index.html [hunk 7]
```diff
@@ -19,7 +19,7 @@
 		<div class="--title">
 			<div class="--category-img"><img src="https://plot.ly/gh-pages/documentation/static/images/python-small.png" alt=""></div>
 			<div class="--body">
-				<h1>Plotly Python Library Statistical Charts</h1>
+				<h1> Library Statistical Charts</h1>
```
Comment (Kully) [inline _posts/python/statistical/2017-02-24-python-statistic-index.html]: why are you removing `Plotly Python` here?

File: all_static/images/Plotly-feed2.html [hunk 8]
```diff
@@ -12381,7 +12381,7 @@ <h4 class="delta heading text--left">The Plotly community</h4>
                 </div><!--
                 --><div class="grid__item one-half">
                     <h4 class="delta heading text--left">Programmer? We've got you covered.</h4>
-                    See our <a href="https://plot.ly/api/" target="_self" class="link link--impt epsilon">API libraries</a><br>
+                    See our <a href="https://plot.ly/api/" target="_self" class="link link--impt epsilon">Open Source Graphing libraries</a><br>
```
Comment (Kully) [inline all_static/images/Plotly-feed2.html]: capital L

File: all_static/images/Plotly.html [hunk 9]
```diff
@@ -12381,7 +12381,7 @@ <h4 class="delta heading text--left">The Plotly community</h4>
                 </div><!--
                 --><div class="grid__item one-half">
                     <h4 class="delta heading text--left">Programmer? We've got you covered.</h4>
-                    See our <a href="https://plot.ly/api/" target="_self" class="link link--impt epsilon">API libraries</a><br>
+                    See our <a href="https://plot.ly/api/" target="_self" class="link link--impt epsilon">Open Source Graphing libraries</a><br>
```
Comment (Kully) [inline all_static/images/Plotly.html]: capital L

File: _posts/pandas/2015-07-19-pandas_index.html [hunk 10]
```diff
@@ -12,7 +12,7 @@
 		<div class="--title">
 			<div class="--category-img"><img src="https://plot.ly/gh-pages/documentation/static/images/pandas-small.png" alt=""></div>
 			<div class="--body">
-				<h1>Plotly Python Library for Pandas</h1>
+				<h1>Plotly Python Open Source Library for Pandas</h1>
```
Comment (Kully) [inline _posts/pandas/2015-07-19-pandas_index.html]: graphing?

**Comment by jdamiba** [comment]
I standardized the use of "Open Source Graphing Libraries" throughout the site. 

In terms of images, does anyone know how i can host images on images.plot.ly? That's where the other images for the homepage are being served from.

**Comment by Kully** [comment]
@jdamiba give me a ping when you've addressed/responded to all my comments. Thanks 😄

**Comment by nicolaskruchten** [comment]
Once you deploy https://github.com/plotly/plotly.github.io/pull/366 you can use that image ;)

**Comment by jdamiba** [comment]
I moved the note to the bottom of the page, along with a hyperlink to https://plot.ly/python/static-image-export/#supported-formats

webglnote2

**Comment by cldougl** [comment]
I'm a bit unsure about adding the rest library in that spot as it is not open-sourced
screen shot 2018-09-04 at 11 58 51 am

**Comment by jdamiba** [comment]
I removed the REST API box. I like the way it looks now a lot since Dash gets a level all to itself. 

screen shot 2018-09-04 at 1 18 31 pm

**Comment by cldougl** [comment]
We may want to consider making the icons per row 3, 2 rather than 4, 1 in the above screenshot though that can be done in a follow up pr

**Comment by nicolaskruchten** [comment]
+1 to the 3/2 vs 4/1 per row.

Other thoughts:

* We want to move away from "Download free and open-source *clients* for:"
* We need to be clear that Dash isn't like the others... it's not another "Open Source Graphing" library per se... App Development Library?

**Comment by nicolaskruchten** [comment]
OK, let's ship it 💃

**Pull request merged**